content/installation/Webhook/_index.md

```diff
@@ -0,0 +1,150 @@
+  --set config.settings.clustername=<CLUSTER_NAME> \
+  --set image.repository=infracloudio/botkube \
+  --set image.tag=v0.10.0 \
+  infracloudio/botkube
+  ```
+
+  {{% /tab %}}
+  {{% tab name="Helm 2" %}}
+
+  ```bash
+  $ helm install --version v0.10.0 --name botkube --namespace botkube \
+  --set config.communications.webhook.enabled=true \
+  --set config.communications.webhook.url=<WEBHOOK_URL> \
+  --set config.settings.clustername=<CLUSTER_NAME> \
+  --set image.repository=infracloudio/botkube \
+  --set image.tag=v0.10.0 \
+  infracloudio/botkube
+  ```
+
+  {{% /tab %}}
+  {{< /tabs >}}
+
+  where,<br>
+  - **WEBHOOK_URL** is an outgoing webook URL to which BotKube will POST the events <br>
+  - **CLUSTER_NAME** is the cluster name set in the incoming messages<br>
+
+     Configuration syntax is explained [here](/configuration).
+
+  {{% notice note %}}
+    With the default configuration, BotKube will watch all the resources in all the namespaces for _create_, _delete_ and _error_ events.<br>
+    If you wish to monitor only specific resources, follow the steps given below:
+  {{% /notice%}}
+
+  - Create new file config.yaml and add resource configuration as described on the [configuration](/configuration) page.
+
+    (You can refer sample config from https://github.com/infracloudio/botkube/tree/develop/helm/botkube/sample-res-config.yaml)
+
+    ```
+    config:
+      ## Resources you want to watch
+      resources:
+        - name: pod                # Name of the resources e.g pod, deployment, ingress, etc.
+          namespaces:              # List of namespaces, "all" will watch all the namespaces
+            include:
+              - all
+            ignore:
+              - kube-system
+          events:                  # List of lifecycle events you want to receive, e.g create, update, delete, error OR all
+            - create
+            - delete
+            - error
+        - name: job
+          namespaces:
+            include:
+              - all
+            ignore:
+              -
+          events:
+            - create
+            - update
+            - delete
+            - error
+          updateSetting:
+            includeDiff: true
+            fields:
+              - spec.template.spec.containers[*].image
+              - status.conditions[*].type
+    ```
+  - Pass the yaml file as a flag to `helm install` command.
+    e.g
+
+    ```
+    helm install --version v0.10.0 --name botkube --namespace botkube -f /path/to/config.yaml --set=...other args..
+    ```
+
+  Alternatively, you can also update the configuration at runtime as documented [here](/configuration/#updating-the-configuration-at-runtime)
+
+
+#### Using kubectl
+
+- Make sure that you have kubectl cli installed and have access to Kubernetes cluster
+- Download deployment specs yaml
+
+```bash
+$ wget -q https://raw.githubusercontent.com/infracloudio/botkube/master/deploy-all-in-one.yaml
```

Comment:
Should we mention the stable version here as well?
Applies to both in install/uninstall command and all rest of installations.

Reply:
Sounds good!